karrot/activities/serializers.py

```diff
@@ -181,13 +182,17 @@ class Meta:
         ]
 
     participants = serializers.SerializerMethodField()
+    feedback_dismissed_by = serializers.SerializerMethodField()
     feedback_due = DateTimeFieldWithTimezone(read_only=True, allow_null=True)
 
     date = DateTimeRangeField()
 
     def get_participants(self, activity):
         return [c.user_id for c in activity.activityparticipant_set.all()]
 
+    def get_feedback_dismissed_by(self, activity):
+        return [c.user_id for c in activity.activityparticipant_set.all() if c.feedback_dismissed]
```

Comment:
Maybe worth a comment to explain why the filtering is being done in python, rather than the db, we can borrow the comment used elsewhere:

_we are filtering in python to make use of prefetched data_